dev/tidb-in-kubernetes/deploy/gcp-gke.md

```diff
@@ -187,12 +187,6 @@ mysql -h <tidb_ilb_ip> -P 4000 -u root
     kubectl --kubeconfig credentials/kubeconfig_<cluster_name> get po -n tidb
     ```
 
-    {{< copyable "shell-regular" >}}
-
-    ```bash
-    helm --kubeconfig credentials/kubeconfig_<cluster_name> ls
```

Comment:
helm doesn't have the `--kubeconfig` flag.

Reply:
what's your helm version?

```
$ helm --help | grep kubeconfig
      --kube-context string             name of the kubeconfig context to use
      --kubeconfig string               absolute path to the kubeconfig file to use
$ helm version -c 
Client: &version.Version{SemVer:"v2.14.3", GitCommit:"0e7f3b6637f7af8fcfddb3d2941fcc7cbebb0085", GitTreeState:"clean"}
```

Reply:
helm version `v2.9.1`

```
$ helm --help | grep kubeconfig
      --kube-context string             name of the kubeconfig context to use
```

Reply:
This section should be reserved.

Reply:
But it can't work on helm `v2.9.1`

Reply:
`--kubeconfig` is introduced in 2.10.0: https://github.com/helm/helm/releases/tag/v2.10.0. we can remove this section because the minimum version requirement in this docs is `2.9.0`. Alternatively, we can add a note here that `--kubeconfig` requires helm 2.10.0+. What do you prefer?

Reply:
> `--kubeconfig` is introduced in 2.10.0: https://github.com/helm/helm/releases/tag/v2.10.0. we can remove this section because the minimum version requirement in this docs is `2.9.0`. Alternatively, we can add a note here that `--kubeconfig` requires helm 2.10.0+. What do you prefer?

I think adding a note makes more sense. The `helm` version requirement in this document is >= 2.9.0 and < 3.0.0, so technically it is not wrong to put `helm --kubeconfig` here. Just need to remind users to upgrade `helm` of a lower version in order to use this command.

Reply:
Can we just update the version requirement to >=2.10.0? It's already 2.14.x now.

Reply:
added a note

@yikeke @DanielZhangQD @cofyc PTAL